ssm doesn't work on java > 8

The resources are not properly loaded which leads to nullpointer exceptions.
I've tested this on linux and windows. With java 8 it does work correctly.

This has probably something to do with the way resources are retrieved in newer java version.
The stack trace is as follows:

```
java.lang.NullPointerException
        at nl.uu.cs.ssmui.HelpFromProp.getProps(HelpFromProp.java:41)
        at nl.uu.cs.ssmui.HelpFromProp.<init>(HelpFromProp.java:56)
        at nl.uu.cs.ssmui.SSMRunner.<init>(SSMRunner.java:165)
        at nl.uu.cs.ssmui.Runner.<init>(Runner.java:37)
        at nl.uu.cs.ssmui.Runner.main(Runner.java:157)
```

If I fix that exception by putting in a bogus one (the help text will be gone then)
I get the following error:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at nl.uu.cs.ssmui.SSMRunner.initComponents(SSMRunner.java:447)
        at nl.uu.cs.ssmui.Runner.<init>(Runner.java:38)
        at nl.uu.cs.ssmui.Runner.main(Runner.java:157)
Caused by: java.lang.NullPointerException
        at java.desktop/javax.swing.ImageIcon.<init>(ImageIcon.java:217)
        at nl.uu.cs.ssmui.Images.getImage(Images.java:35)
        at nl.uu.cs.ssmui.Images.<clinit>(Images.java:40)
        ... 3 more
```

If I find the time I will try to fix it myself and report back.

--- a/src/nl/uu/cs/ssmui/HelpFromProp.java
+++ b/src/nl/uu/cs/ssmui/HelpFromProp.java
@@ -34,7 +34,7 @@ public class HelpFromProp
 	
 	private static Properties getProps( String nm )
 	{
-		URL url = nm.getClass().getResource( "/Help/" + nm + ".prop" ) ;
+		URL url = HelpFromProp.class.getResource( "/Help/" + nm + ".prop" ) ;
 		Properties props = new Properties() ;
 		try
 		{

--- a/src/nl/uu/cs/ssmui/Images.java
+++ b/src/nl/uu/cs/ssmui/Images.java
@@ -32,7 +32,7 @@ public class Images
 	
 	private static ImageIcon getImage( String nm )
 	{
-		return new ImageIcon( nm.getClass().getResource( "/Images/" + nm + ".gif" ) ) ;
+		return new ImageIcon( Images.class.getResource( "/Images/" + nm + ".gif" ) ) ;
 	}
 	
 	static

